### DISCUSSION TITLE
Using --install-cert for multiple locations

### DISCUSSION THREAD

**Discussion by Chouhada** [acmesh-official/acme.sh#5791] [Q&A]
Hi
What is the appropriate way to use `--install-cert` for multiple applications?

For example, I have a setup where I want to place the certs to 2 locations and run different reload commands.
Individually, I have these commands working. However when running `acme.sh --renew -d DOMAIN.com --force` I only see the output for whatever the last `--install-cert` was executed. As a result, when the automatic renewal period comes around, I think only one will get executed.

Could someone clarify how to approach this, or even if I'm thinking about this incorrectly.
Many thanks

**Comment by neilpang** [comment]
use `--install-cert  --reloadcmd /my.copy.sh`

use your scrip to copy to anywhere.
